### ISSUE TITLE
Project-search

### ISSUE THREAD

**Issue by VicenteMerino** [IIC2413/Syllabus-2019-2#194]
Hola, cuando uso el project-search, lo que tengo que entregar en definitva son aquellas llaves que tengan sender o receiver el nombre del proyecto?

Gracias

**Comment by nivek0o0** [comment]
Sí, todos los correos enviados o recibidos por el proyecto entregado en la ruta. Por ejemplo:
`/project-search?nombre=MineraDelSol`

**Comment by tomcan127** [comment]
Hola,
Puedo usar el formato de los otras rutas? 
` /messages/project-search/nombre`

**Comment by nivek0o0** [comment]
@tomcan127 
No, la idea de entregarles rutas bien definidas es para estandarizar la corrección.
Además buscamos evaluar distintas formas de hacer requests.
